iocage_lib/ioc_plugin.py

```diff
@@ -1429,9 +1427,6 @@ def __check_manifest__(self, plugin_conf):
                 },
                 _callback=self.callback)
 
-        if write:
-            self.json_write(plugin_conf)
```

Comment:
Wondering why this changed?

Reply:
I don't know why it was there in the first place :p ioc plugin has no function with that signature